src/routes/+page.svelte

```diff
@@ -1,9 +1,11 @@
 <script lang="ts">
-	import FeatureCard from './feature_card.svelte';
-	import BigNumberCard from './big_number_card.svelte';
-	import ProblemCard from './problem_card.svelte';
-	import Logo from '$lib/components/Logo.svelte';
-	import LinkedinLogo from '$lib/assets/linkedin_logo.svelte';
+	import FeatureCard from './feature_card.svelte'
+	import BigNumberCard from './big_number_card.svelte'
+	import ProblemCard from './problem_card.svelte'
+	import Logo from '$lib/components/Logo.svelte'
+	import LinkedinLogo from '$lib/assets/linkedin_logo.svelte'
+	import Pill from '$lib/components/Pill/Pill.svelte'
+	import AnimatedCarousel from '$lib/components/AnimatedCarousel/AnimatedCarousel.svelte'
```

Comment:
I was wondering if we could possibly just move both of these files to:

* `src/lib/components/Pill.svelte`
* `src/lib/components/AnimatedCarousel.svelte`

I'm assuming having them in their own folder would lead to better organization if they had other co-located files for logic or components, but since these are single files do we still need to have them in their own folder? 🤔

Reply:
the idea is to have in the same folder all the files for it, as an example, for Pill, the Pill folder would have:
- `Pill.svelte`
- `Pill.stories.svelte` for storybook
- `Pill.spec.svelte` for testing
- and potentially a `__snapshots__` folder for snapshot tests of these components to make sure nothing changed.

Reply:
Sounds good to me then ✅